wire-ios-sync-engine/Tests/Source/Synchronization/ZMOperationLoopTests.m

```diff
@@ -42,12 +42,17 @@ - (void)setUp
     self.mockRequestStrategy = [[MockRequestStrategy alloc] init];
     self.mockUpdateEventProcessor = [[MockUpdateEventProcessor alloc] init];
     self.mockRequestCancellation = [[MockRequestCancellation alloc] init];
+
+    LastEventIDRepository *lastEventIDRepository = [[LastEventIDRepository alloc] initWithUserID:self.userIdentifier
+                                                                                    userDefaults:NSUserDefaults.standardUserDefaults];
```

Comment:
same

Reply:
fixed in c8a59a522c5b2a06d5a9bf871a2a7e174175247d